src/components/node.js

```diff
@@ -392,6 +392,7 @@ class Node extends React.Component<INodeProps, INodeState> {
         ref={this.nodeRef}
         opacity={opacity}
         transform={`translate(${x}, ${y})`}
+        style={{ transform: `matrix(1, 0, 0, 1, ${x}, ${y})` }}
```

Comment:
This fixes a bug with asynchronous node rendering. Chrome or d3 will randomly add a transform property with the incorrect properties.